src/app/app.js

```diff
@@ -214,3 +179,7 @@ export default class App {
     return true;
   }
 }
+
+App.prototype.pay = deprecate(function pay(amount, destination = null) {
```

Comment:
Let's not touch App's prototype here. I suggest using regular class method with `@deprecated` tag comment instead. http://usejsdoc.org/tags-deprecated.html

I mean something like 

```js
/**
  * @deprecated Please use `Mobius.App.charge()` instead." 
  * @param {number} amount - payment amount
  * @param {?string} destination - third party receiver address
  * @returns {StellarSdk.Operation} payment operation
*/
pay = deprecate((amount, destination = null) => {
  return this.charge(amount, destination)
}, '...')
```

My point here is that, since "deprecated" is just a status, there is no need to declare it separately, especially, if we're losing method documentation.

Reply:
Fixed, please check if I got it right this time.